homeassistant/components/fritzbox_callmonitor/sensor.py

```diff
@@ -148,8 +150,9 @@ async def async_added_to_hass(self):
         )
         self._monitor.connect()
 
-    async def async_will_remove_from_hass(self):
+    async def async_will_remove_from_hass(self, *args: Any) -> None:
```

Comment:
This is not the correct interface. `async_will_remove_from_hass` is a standard Entity method.
```suggestion
    async def async_will_remove_from_hass(self) -> None:
```